packages/amplify-cli-core/src/utils/open.ts

```diff
@@ -0,0 +1,16 @@
+import open from 'open';
+import ciInfo from 'ci-info';
+import { ChildProcess } from 'child_process';
+
+/**
+ * Helper function to Open stuff like URLs, files, executables. Cross-platform and opens only if its run in non-ci environmets
+ * This is wrapper for https://github.com/sindresorhus/open
+ * @param target The thing you want to open. Can be a URL, file, or executable.
+ * @param options
+ */
+export const openIfNotCI = (target: string, options: open.Options): Promise<ChildProcess | void> => {
```

Comment:
a little cleaner to write:
```ts
export const openIfNotCI = ciInfo.isCI ? async () => {} : open
```

Reply:
I think this is less readable than what we have and would stick to it